parallel-install/example/example.go

```diff
@@ -150,6 +145,22 @@ func main() {
 	if err != nil {
 		log.Fatalf("Failed to uninstall Kyma: %v", err)
 	}
+
+	postUninstallerCfg := postuninstaller.Config{
+		Log:              installationCfg.Log,
+		KubeconfigSource: installationCfg.KubeconfigSource,
```

Comment:
can't `commonRetryOpts` be part of this configuration? then `NewPostUninstaller` would have the same signature , maybe they can even use the same config

Reply:
Ultimately I will get rid of the introduced struct and merge it with `Deletion`, so input will no longer be needed.